app/upgrades.go

```diff
@@ -22,34 +23,60 @@ func (app UmeeApp) RegisterUpgradeHandlers(experimental bool) {
 		panic(err)
 	}
 
-	app.registerV3_0Upgrade(upgradeInfo)
-	app.registerV3_1Upgrade(upgradeInfo)
-	app.registerV3_2Upgrade(upgradeInfo)
+	app.registerV3_0_Upgrade(upgradeInfo)
+	app.registerV3_1_Upgrade(upgradeInfo)
+
+	app.registerV3_1to3_3_Upgrade(upgradeInfo)
+	app.registerV3_2to3_3_Upgrade(upgradeInfo)
+}
+
+// performs upgrade from v3.1 -> v3.3 (including the v3.2 chanages)
+func (app *UmeeApp) registerV3_1to3_3_Upgrade(_ upgradetypes.Plan) {
+	const planName = "v3.1-v3.3"
+	app.UpgradeKeeper.SetUpgradeHandler(
+		planName,
+		func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			ctx.Logger().Info("Upgrade handler execution", "name", planName)
+			ctx.Logger().Info("Run v3.3 migrator")
+			err := upgradev3_3.Migrator(app.GovKeeper, app.interfaceRegistry)(ctx)
+			if err != nil {
+				return fromVM, err
+			}
+			ctx.Logger().Info("Run x/bank v0.46.5 migration")
+			err = bankkeeper.NewMigrator(app.BankKeeper).Migrate3_V046_4_To_V046_5(ctx)
+			if err != nil {
+				return fromVM, err
+			}
+			ctx.Logger().Info("Run module migrations")
+			return app.mm.RunMigrations(ctx, app.configurator, fromVM)
+		})
 }
 
-// performs upgrade from v3.1 -> v3.2
-func (app *UmeeApp) registerV3_2Upgrade(_ upgradetypes.Plan) {
-	const planName = "v3.2"
+// performs upgrade from v3.2 -> v3.3
+func (app *UmeeApp) registerV3_2to3_3_Upgrade(_ upgradetypes.Plan) {
+	const planName = "v3.3"
```

Comment:
maybe `"v3.2-v3.3"` for consistency

Reply:
In general for direct migration (`y.x -> y.(x+1)`) we were using `y.(x+1)` name. 
But I don't have strong preference, we can use `"v3.2-v3.3"`

Reply:
updated to "v3.2-v3.3"